android/src/main/java/com/lwansbrough/RCTCamera/RCTCamera.java

```diff
@@ -109,6 +114,36 @@ public int getPreviewHeight(int type) {
         return result;
     }
 
+    public Camera.Size getBestVideoSize(int type, int width, int height)
+    {
+        Camera camera = _cameras.get(type);
+        Camera.Size result = null;
+        if(camera == null) {
+            return null;
+        }
+        Camera.Parameters params = camera.getParameters();
+        // defer to preview instead of params.getSupportedVideoSizes() http://bit.ly/1rxOsq0
+        List<Size> sizes = params.getSupportedPreviewSizes();
+
```

Comment:
`params.getSupportedVideoSizes();` should be preferred! (pull request coming up ;))